### ISSUE TITLE
Create Companion tutorial

### ISSUE THREAD

**Issue by fcheadle** [thesadrogue/TheSadRogue.Integration#7]
The integration library should contain a very simple tutorial to help people get familiar with the concepts of GoRogue, SadConsole, and the Integration library.

The tutorial should include several parts:

1. Getting started - create a blank map, put a character on the screen, and move the character around.
2. Generating a more in-depth map
3. FOV & Exploration
4. Creatures & Melee Combat
5. Messages & Look
6. Items & Inventory
7. Ranged Combat
8. Save & Main Menu

**Comment by Chris3606** [comment]
@Thraka had potential plans into replicating something similar to the [famous Python-libtcod tutorial](http://www.roguebasin.com/index.php?title=Complete_Roguelike_Tutorial,_using_python%2Blibtcod) using the integration library; that may fit into this vein?

**Comment by fcheadle** [comment]
Yeah, that article is where I got this list from. It's slightly altered because some things are made trivially easy by the integration lib, and I think that some things in the tutorial are presented out-of-order.

**Comment by Thraka** [comment]
I'm using http://rogueliketutorials.com/ which is the python 3 version of that tutorial.

**Comment by fcheadle** [comment]
I'm writing the `map generation` piece as we speak, and will post it here shortly
